src/VectorizedSolver.cpp

```diff
@@ -246,6 +240,32 @@ void VectorizedSolver::initializeSourceArrays() {
 
 
 /**
+ * @brief Initializes the FSR volumes and Materials array.
+ */
+void VectorizedSolver::initializeFSRs() {
+
+  CPUSolver::initializeFSRs();
+
+  /* Compute the number of SIMD vector widths needed to fit energy groups */
+  _num_vector_lengths = (_num_groups / VEC_LENGTH) + 1;
+
+  /* Reset the number of energy groups by rounding up for the number
+   * of vector widths needed to accomodate the energy groups */
+  _num_groups = _num_vector_lengths * VEC_LENGTH;
+  _polar_times_groups = _num_groups * _num_polar;
+
+  /* Allocate array of mutex locks for each FSR */
+  _FSR_locks = new omp_lock_t[_num_FSRs];
```

Comment:
`_FSR_locks` is already allocated and initialized within `CPUSolver::initializeFSRs();` so this can be removed.